streaming/base/shared/prefix.py

```diff
@@ -113,6 +114,11 @@ def _check_and_find(streams_local: list[str], streams_remote: list[Union[str, No
 
     for prefix_int in _each_prefix_int():
 
+        if prefix_int >= MAX_PREFIX_INT:
+            raise ValueError(f'prefix_int exceeds {MAX_PREFIX_INT}. This may happen ' +
+                             f'when you mock os.path.exists or os.stat so the filelock ' +
+                             f'checks always returns ``True`` ' + f'you need to clean up TMPDIR.')
```

Comment:
Does the sentence end after ``True``? A bit confused what the end of this message is saying

(minor nit)
```suggestion
                             f'checks always returns `True` ' + f'you need to clean up TMPDIR.')
```